Katello/katello
# Fixes #35783 - Remove old Pulp 2 docker port setting

#### What are the changes introduced in this pull request?
Removes the old `pulp_docker_registry_port` setting from Pulp 2. Also cleans up some of the related Pulp 2 methods.

#### Considerations taken when implementing this change?

This should not affect any of the Pulp 3 container workflows.

#### What are the testing steps for this pull request?
1) Edit the `pulp_docker_registry_port` setting
2) Patch in this PR and run the DB migrations
3) Check that the setting is gone
4) Check that there are no traces of the setting in the database
5) Check that there are no traces of the setting in the code
6) Double check that container smart proxy syncing still works (and thus repo syncing too)
  -> And `podman pull` from a client
